### ISSUE TITLE
Workflow Penggunaan Tea Module Installer Membingungkan

### ISSUE THREAD

**Issue by dikyarga** [tealinuxos/tea-module-installer#4]
#IMHO Aplikasi ini idenya bagus, membantu _programmer_ pemula memasang aplikasi IDE, _compiler_ dan _tools_ lainya dengan mudah melalui GUI. Namun, ketika pengguna pertama kali menggunakanya, kebanyakan akan mengalami kebingung-an, seperti apa yang harus dilakukan pertama kali ? mana yang harus di klik dan lain-lain, kerena _user interface_ -nya kurang intuitif. 

Ada ide kah agar pengguna baru dapat menggunakanya tanpa perlu dibimbing ?

summon : @rezafaizarahman :smiley:  Hehe

**Comment by rezafaizarahman** [comment]
Duh, belum pernah nyoba yang versi terbaru 😂  

Tapi kalau modelnya sama seperti yang dulu (install IDE satu-satu) dan itu kurang intuitif, mungkin nanti bisa dikembangin pakai model paket. One click installer. 

.
Want to be a Front End developer?
- Click paket Front End Developer
- Install (otomatis) paket-paket (git, atom dll)
- Tunggu pemasangan
- Trus tinggal pakai dah 😁

**Comment by dikyarga** [comment]
Sebenarnya yang sekarang sih udah di buat modul-modul gitu, Mas. Ada 3, Pemprograman, Multimedia dan Jaringan. Seperti ini : ![tea-linux-module-installer-look-like](https://cloud.githubusercontent.com/assets/6753674/21318546/383c4164-c63c-11e6-95f4-059028b45919.png) Tapi karena kita mau fokus buat programmer aja, saran mas @rezafaizarahman bagus juga. Jadi nanti ada _bundle_ Back-end, Front-end, DevOps, dll. Tapi tetep bisa di custom lagi isi paketnya ya, Mas. 

Nah yang masih kita bingung adalah alurnya, Mas. Yang sekarang 3 opsi paket / _bundle_ itu di _sidebar_ mas.

Ada saran untuk UI & UX _flow_ -nya, Mas @rezafaizarahman ?

**Comment by rezafaizarahman** [comment]
Iya. Kalau dari screenshot di atas belum masuk, Dik. Sidebar jadinya cuma sekedar kategori pemisah dari masing-masing module, bukan bundle. Karena setelah milih programming, user masih harus cari aplikasi mana yang harus mereka pasang. 

Maksudku kaya [ini](https://github.com/thoughtbot/laptop). Kita cuma jalanin sekali langsung install keseluruhan tools.

Flownya pilih bundle dulu, custom (optional), install, pakai. 

Untuk tipe devnya ada Full-Stack, Front-End, Back-End, Mobile, Students. Kalau boleh ada bundle Designer 😁. 

Anyway, ada referensi bagus soal development http://stackoverflow.com/research/developer-survey-2016#technology-development-environments

**Comment by mnirfan** [comment]
Setelah salah satu kategori diklik langsung kepilih semua yang berhubungan mas. Kalau mau ada yg ngga diinstall tinggal dihilangkan seleksinya.

**Comment by dikyarga** [comment]
Barusan ngobrol langsung dengan Mas @rezafaizarahman , idenya keren banget. Mirip tool [Laptop](https://github.com/thoughtbot/laptop) dengan kemasan yang lebih menarik dan beralur.

Tea Module Installer ini seperti kita butuh ubah namanya, kerana kita bakal buat aplkasi ini bisa jalan di semua OS yang berbasis Ubuntu pake [Electron](http://electron.atom.io/). Ini juga salah satu cara ngajak pengguna  distro lain buat pindah ke TeaLinuxOS dengan memberi "iming-iming" fitur lebih jika aplikasi ini dipakai langsung di TeaLinuxOS.

Bagaimana menurutmu, @mnirfan ?

**Comment by dikyarga** [comment]
Berelasi dengan #3

**Comment by mnirfan** [comment]
Ditunggu penjelasan lengkapnya. Sepertinya juos :D

**Comment by dikyarga** [comment]
Aku udah di _camp_ [Doscom](http://doscom.org) ini :smile: @mnirfan
